src/go/k8s/controllers/redpanda/cluster_controller.go

```diff
@@ -142,6 +140,16 @@ func (r *ClusterReconciler) Reconcile(
 		}
 	}
 
+	// Ensure StatefulSet #replicas equals cluster requirement.
+	if sts.Spec.Replicas != redpandaCluster.Spec.Replicas {
+		sts.Spec.Replicas = redpandaCluster.Spec.Replicas
+		err = r.Update(ctx, &sts)
+		if err != nil {
+			log.Error(err, "Failed to update StatefulSet", "StatefulSet.Namespace", sts.Namespace, "StatefulSet.Name", sts.Name)
```

Comment:
does this pass linting rules? 

```
>>> len("                       log.Error(err, \"Failed to update StatefulSet\", \"StatefulSet.Namespace\", sts.Namespace, \"StatefulSet.Name\", sts.Name)")                                             
119
```

in python it yields 119; we break at 80 chars.

Reply:
I do run `crlfmt -wrap=80 -w .` locally and the output includes the long line you noticed. We also run the same command as a Check (with no difference).

I haven't found a solution or whether this is expected behavior somehow. I updated the last commit message to reflect this issue. The Cockroach's coding style mentions their default tab space is 2 instead of 4 chars, (which is ours) but this doesn't sound like the reason. https://wiki.crdb.io/wiki/spaces/CRDB/pages/181371303/Go+coding+guidelines

After your comment, however, I did find we're not running `golangci-lint` as a check. I'll see how we can add this independently of this PR - I did run it locally.